docs/docs/governance/unlock-dao-tokens.mdx

```diff
@@ -119,3 +119,91 @@ At that point, the collected fees can be "swapped and burned" by calling the `sw
 ### Swapping UDT for UP
 
 If you have earn UDT on any network, you can swap them for UP. In order to do so, you first need to "bring" the UDT back to Base (using a bridge like [Superbridge](https://superbridge.app/base)), and then swap them for UP. The migration is a 1:1000 swap and can be conveniently achieved through a [migration website](https://up-swap.unlock-protocol.com/).
+
+## What is the Unlock Protocol UP token (UP)?
+
+The **UP token** (Unlock Protocol Token) is the native governance token of the Unlock Protocol.
+
+## What is the Unlock Protocol UP token used for?
+
+The UP token serves several key functions:
+
+1. **Incentives**: The UP token is used to reward developers and creators who build on or integrate the Unlock Protocol. For example, developers who deploy smart contracts on the platform may earn UP tokens as rewards.
+2. **Governance**: UP token holders can participate in the governance of the protocol by voting on proposals or protocol changes.
+
+## How do you get the Unlock Protocol UP token?
+
+There are multiple ways for individuals to get the UP token.
+
+- **Swap UDT for UP** — Holders of UDT can swap UDT for UP
+- **Purchase UP directly** — The Unlock Protocol UP token may be acquired on decentralized exchanges such as Uniswap
+- **Receive UP as grants from the Unlock DAO** — The Unlock DAO has a grants program, where builders can request grants of UP for projects built using Unlock Protocol
+- **Protocol rewards** — Transactions that use Unlock Protocol that contain a financial component can include a referrer fee, paid in the Unlock Protocol UP token
```

Comment:
Please add link to relevant section above

Reply:
will add links as part of larger linking work